parachain/pallets/inbound-queue/src/test.rs

```diff
@@ -162,16 +165,16 @@ fn expect_events(e: Vec<RuntimeEvent>) {
 	assert_eq!(last_events(e.len()), e);
 }
 
-pub fn new_tester(gateway: H160) -> sp_io::TestExternalities {
-	new_tester_with_config(inbound_queue::GenesisConfig { gateway })
+pub fn new_tester<T: Config>(gateway: H160) -> sp_io::TestExternalities {
```

Comment:
Previously I add `owner` to genenis config which required `T: Confg`, but since the default owner account from Parity team may not the one as we want, IMO account from our team make more sense here so remove it.